my-app/src/components/Card/Card.jsx

```diff
@@ -6,16 +6,13 @@ const Card = (props) => {
   return (
     <div className="card-wrapper">
       <div className="from-to">
-        <Avatar />
-        <div className="sender-name">Eric Koslow</div>
+        <Avatar url={props.element.senderAvaURL} />
+        <div className="sender-name">{props.element.senderName}</div>
         <div className="to">TO</div>
-        <Avatar />
-        <div className="receiver-name">Madison Kochenderfer</div>
-      </div>
-      <div className="message">
-        Great job on that presentation to the Board yesterda' everybody was
-        really impressed!
+        <Avatar url={props.element.receiverAvaURL} />
+        <div className="receiver-name">{props.element.receiverName}</div>
       </div>
+      <div className="message">{props.element.message}</div>
```

Comment:
We can add extra safety here by using `.?` during props chaining: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Like `props.?element.?message`

Reply:
nit: As we use only `element` field from props, we can use props destructuring in this component: 
```
const Card = ({ element }) => {
```
and then use just `element` variable. Not a big deal thou, it depends more on the project code style.

By the way, it will be better to rename `element` in to more clear way, for instance `message`.